app/experimenter/docs/swagger-ui.html

```diff
@@ -3551,11 +3551,11 @@
           },
           "startDate": {
             "type": "string",
-            "readOnly": true
+            "format": "date-time"
```

Comment:
Are these no longer readOnly then?

Reply:
The whole V6 API is read only, it only responds to GET requests:

https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/api/v6/views.py#L11-L12